cdi/pom.xml

```diff
@@ -20,17 +20,22 @@
     </dependency>
 
     <dependency>
-      <groupId>javax.servlet</groupId>
-      <artifactId>javax.servlet-api</artifactId>
+      <groupId>jakarta.servlet</groupId>
+      <artifactId>jakarta.servlet-api</artifactId>
       <scope>provided</scope>
     </dependency>
 
-    <dependency>
-      <groupId>javax.enterprise</groupId>
-      <artifactId>cdi-api</artifactId>
-      <version>2.0.SP1</version>
-      <scope>provided</scope>
-    </dependency>
+      <dependency>
```

Comment:
1) jakarta.enterprise.cdi-api:4.0.1 should be used instead
2) it gives transitive dependency for jakarta.annotation-api as well making it not necessary to define it explicitly